### ISSUE TITLE
Allow flexible default solar flux/mag in reflectance equivalency

### ISSUE THREAD

**Issue by jianyangli** [NASA-Planetary-Science/sbpy#148]
At this moment the default solar flux and magnitude are hard coded.  But @mkelley suggested that they should be allowed to change, both in runtime and in package configuration.  Something like:

```
import numpy as np
import astropy.units as u
from sbpy.units import VEGAmag, reflectance

mag = 3.4 * VEGAmag
xsec = np.pi * (460 * u.km)**2
with default_solar_flux(f_sun, M_sun):
    ref = mag.to('1/sr', reflectance(cross_section=xsec))
```

**Comment by jianyangli** [comment]
Being address in PR #172

**Comment by mkelley** [comment]
Addressed with #186 .
